src/web/routes/application/address/select-address/select-address.js

```diff
@@ -55,18 +58,22 @@ const behaviourForPost = () => (req, res, next) => {
   next()
 }
 
+const contentSummary = req => path(['session', 'claim', 'selectedAddress'], req) ? addressContentSummary(req) : null
```

Comment:
There seems to be an inconsistency with how you've set the contentSummary in the two pages, could/should we be consistent and use `isNavigable` for both? This line confused me...

Reply:
There is no isNavigable function for this page. It's always shown if it's toggled on.